apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -43,6 +81,92 @@ ConversationView.prototype = {
     return this._switchToContactsPageObject();
   },
 
+  openParticipants: function() {
+    this.accessors.headerTitle.tap();
+
+    var ParticipantsView = require('../participants/view');
+    var participantsView = new ParticipantsView(this.client);
+    participantsView.accessors.waitToAppear();
+    return participantsView;
+  },
+
+  openReport: function(messageId) {
+    var message = this.accessors.findMessage(messageId);
+
+    // Show context menu on message bubble. "1" is the waiting time between
```

Comment:
maybe easiest and most readable is to put "/* sec */" in the longPress call, after `1`. Then you don't even need to describe the waiting time in this comment as this is quite self-describing